lib/notifications/test/docs_example_test.dart

```diff
@@ -21,7 +23,7 @@ void main() {
         );
       },
       executeActionRequest: (request) {
-        print(request.stringToPrint);
+        log(request.stringToPrint);
       },
     );
```

Comment:
Well the thing is this is the code example for the documentation of the `PushNotificationActionHandler`.  
So I guess the example in the doc also needs to be updated, altough I think it would be more confusing to use log instead of print in the example.  
So maybe just add `ignore_for_file: lint_name` to the top of this file?

Reply:
Yes, makes sense. I added it in https://github.com/SharezoneApp/sharezone-app/pull/423/commits/1f6b5c772f55780f74de976ab65c35baaafbb9d9